doc/isorms.xml

```diff
@@ -266,3 +266,65 @@ gap> ImagesElm(map, RMSElement(R, 1, (2, 8), 2));
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="ReesZeroMatrixSemigroupCanonicalLabelling">
+  <ManSection>
+    <Attr Name = "ReesZeroMatrixSemigroupCanonicalLabelling"
+      Label = "for a Rees zero matrix semigroup" Arg = "S"/>
+    <Returns>
+      A matrix with entries from a group or 0.
+    </Returns>
+
+    <Description>
+      If <A>S</A> is a Rees zero matrix group then
+      <C>ReesZeroMatrixSemigroupCanonicalLabelling</C> returns a matrix <C>M</C>
+      with entries in <C>G</C> - the <Ref Attr="IsUnderlyingSemigroup"/>
+      of <A>S</A>, such that the Rees zero matrix semigroup defined by
+      <C>ReesZeroMatrixSemigroup(G, M)</C> is isomorphic to <A>S</A>. Furthermore
+      the output <C>M</C> is canonical in the sense that for any two inputs
+      which are isomorphic Rees zero matrix semigroups the output of this function
+      is the same. <P/>
+
+      <Example><![CDATA[
+gap> S := ReesZeroMatrixSemigroup(SymmetricGroup(3),
+> [[(), (1, 3, 2)], [(), ()]]);;
+gap> ReesZeroMatrixSemigroupCanonicalLabelling(S);
+[ [ (), () ], [ (), (1,2,3) ] ]
+gap> T := ReesZeroMatrixSemigroup(SymmetricGroup(3),
+> [[(1, 2, 3), ()], [(), ()]]);;
+gap> ReesZeroMatrixSemigroupCanonicalLabelling(T);
+[ [ (), () ], [ (), (1,2,3) ] ]
+gap> IsIsomorphicSemigroup(S, T);
+true]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="CanonicalReesZeroMatrixSemigroup">
+  <ManSection>
+    <Attr Name = "CanonicalReesZeroMatrixSemigroup" 
+      Label = "for a Rees zero matrix semigroup" Arg = "S"/>
+
+    <Returns>
+      A Rees zero matrix semigroup.
+    </Returns>
+
+    <Description>
+      This returns the Rees (zero) matrix semigroup <C>T</C> which has the same 
```

Comment:
Does this work for Rees matrix semigroups too?

Reply:
Yes. I accidentally removed that functionality but I added it back. There's no special method it just uses the rees 0-matrix semigroup method and converted back and forth from non 0 rees matrix semigroups.